dali/operators/reader/loader/coco_loader.h

```diff
@@ -24,53 +24,127 @@
 #include "dali/operators/reader/loader/file_label_loader.h"
 #include "dali/core/common.h"
 #include "dali/core/error_handling.h"
+#include "dali/core/geom/vec.h"
 
 namespace dali {
 
 using ImageIdPairs = std::vector<std::pair<std::string, int>>;
+
+inline bool OutPolygonMasksEnabled(const OpSpec &spec) {
+  return spec.GetArgument<bool>("polygon_masks") ||
+    (spec.HasArgument("masks") && spec.GetArgument<bool>("masks"));
+}
+
+inline bool OutPixelwiseMasksEnabled(const OpSpec &spec) {
+  return spec.GetArgument<bool>("pixelwise_masks");
+}
+
+inline bool OutImageIdsEnabled(const OpSpec &spec) {
+  return spec.GetArgument<bool>("image_ids") ||
+    (spec.HasArgument("save_img_ids") && spec.GetArgument<bool>("save_img_ids"));
+}
+
+inline bool HasPreprocessedAnnotations(const OpSpec &spec) {
+  return spec.HasArgument("preprocessed_annotations") ||
+    (spec.HasArgument("meta_files_path") && spec.GetArgument<bool>("meta_files_path"));
+}
+
+inline bool HasSavePreprocessedAnnotations(const OpSpec &spec) {
+  return spec.HasArgument("save_preprocessed_annotations") ||
+    (spec.HasArgument("dump_meta_files") && spec.GetArgument<bool>("dump_meta_files"));
+}
+
+inline bool HasSavePreprocessedAnnotationsDir(const OpSpec &spec) {
+  return spec.HasArgument("save_preprocessed_annotations_dir") ||
+    (spec.HasArgument("dump_meta_files_path") && spec.GetArgument<bool>("dump_meta_files_path"));
+}
```

Comment:
I guess it can be generalized to:
```
inline bool HasFeature(const OpSpec &spec, const string &enabler, const string &enable ) {
  return spec.HasArgument(enabler) ||
    (enable.size() && spec.HasArgument(enable) && spec.GetArgument<bool>(enable ));
}
```